README.md

```diff
@@ -104,15 +104,13 @@ All modules are disabled by default,
 so you'll need to activate them by putting this in your `init.vim` file:
 
 ```lua
-lua <<EOF
```

Comment:
any reason we are removing these? They are easy to copy/past for users.

Reply:
i think its breaking lua hl with treesitter ^^

Reply:
what do you mean? I think the examples should be easy to copy/paste for users.